modules/mvt/docs/api-reference/mapbox-vector-tile-loader.md

```diff
@@ -83,10 +83,12 @@ const geoJSONfeatures = await load(url, MVTLoader);
 
 ## Options
 
-| Option          | Type                                       | Default      | Description                                                                                                                                                                                                                                                                 |
-| --------------- | ------------------------------------------ | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| mvt.coordinates | String                                     | `local`      | When set to `wgs84`, parser will return a flat array of GeoJSON representations of the features with coordinates decoded from provided tile index. When set to `local`, parser will return a flat array of GeoJSON objects with local coordinates decoded from tile origin. |
-| mvt.tileIndex   | Object ({x: number, y: number, z: number}) | _No default_ | Mandatory with `wgs84` coordinates option. An object containing tile index values (x, y, z) to decode WGS84 coordinates.                                                                                                                                                    |
+| Option            | Type                                       | Default      | Description                                                                                                                                                                                                                                                                 |
+| ----------------- | ------------------------------------------ | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mvt.coordinates   | String                                     | `local`      | When set to `wgs84`, parser will return a flat array of GeoJSON representations of the features with coordinates decoded from provided tile index. When set to `local`, parser will return a flat array of GeoJSON objects with local coordinates decoded from tile origin. |
+| mvt.layerProperty | String                                     | `layerName`  | The layer name is added to `feature.properties[layerProperty]` (first adds a `feature.property` object is created if it did not already exist). If set to `null`, it won't add layer name to any property.                                                                  |
```

Comment:
`... it won't add layer name to any property` => `...a layer name property will not be added.`